code-studio/src/js/code-studio.js

```diff
@@ -2,7 +2,56 @@
  * @file Replacement for application.js.erb in dashboard as much as possible.
  *       Code added here will be loaded for (almost) every page in dashboard,
  *       so use sparingly!
+ *
+ * Note: This is included _after_ application.js.erb during our transition period,
+ * so when moving things preserve the include order as much as possible.
  */
 'use strict';
+/* global Sounds */
 
 require('./videos');
+window.React = require('react');
+require('./components/*.jsx', {mode: 'expand'});
```

Comment:
It's a potentially interesting question whether we want to be doing this in the long term. It makes sense as an analogue right now, but eventually I think we want to be explicitly requiring any jsx component we use (and only doing so where we need them). If you agree, maybe worth a TODO here saying that we should eventually replace this with individual requires and get rid of globify.